modules/installation-osp-default-deployment.adoc

```diff
@@ -0,0 +1,47 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+
+[id="installation-osp-default-deployment_{context}"]
+= Resource guidelines for deployment
+
+By default, the {product-title} deployment comprises the following nodes.
+
+== Master nodes
```

Comment:
master nodes  -> control plane machines
worker nodes -> compute machines
bootstrap nodes -> bootstrap machine

Reply:
Done here, but keeping this as a reminder to fix elsewhere.